feature-libs/quote/assets/translations/en/quote.i18n.ts

```diff
@@ -65,15 +65,56 @@ export const quote = {
       newCart: 'New Cart',
       quotes: 'Quotes',
     },
-    confirmSubmitDialog: {
-      title: 'Confirm Send Quote {{ code }}?',
-      name: { heading: 'Name:', content: 'Quote {{ code }}' },
-      description: { heading: 'Description:' },
-
-      warningSubmitNote:
-        'Once a request for quote is submitted it cannot be modified.',
-      confirmSubmitNote: 'Are you sure you want to submit this quote?',
-      confirmSubmitOption: { yes: 'Yes', no: 'No' },
+    confirmActionDialog: {
+      name: { heading: 'Name:', content: '{{ name }}' },
+      description: { heading: 'Description:', content: '{{ description }}' },
```

Comment:
`quote.confirmActionDialog.name.content` is not required - please show the text without translating it.
`quote.confirmActionDialog.description.content` is not required - please show the text without translating it.

Reply:
furthermore it would simplify the structure to 
```
      name: 'Name:',
      description: 'Description:',
```

Reply:
done